provisions/main.yml

```diff
@@ -34,6 +34,13 @@
   tags:
     - saasherder
 
+- name: Configure Jenkins
+  hosts: scanner
+  roles:
+    - api_poll
+  tags:
+    - api_poll
```

Comment:
you added tag here in main.yml for complete role, we dont need tag in individual operations in tasks/main.yml